From 4f6a85ede867782a761b2c40192652ea55221d8f Mon Sep 17 00:00:00 2001
From: MengLi <meng.li@windriver.com>
Date: Wed, 8 Apr 2020 22:39:32 +0800
Subject: [PATCH 21/22] arch: arm64: dts: add compatible string "jedec,spi-nor"

According to upstream commit  de57732da862
("mtd: m25p80: fix module autoloading for "jedec, spi-nor"
and "spi-nor""), it is need to add "jedec,spi-nor" to
flash@xxx dts node, so that mtd spi driver starts up, and
serial spi flash works fine.

Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
index 182ecae80e3d..17f05b94b42f 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
@@ -772,7 +772,7 @@
 		reg = <0>;
 		#address-cells = <1>;
 		#size-cells = <1>;
-		compatible = "micron,n25q256a";
+		compatible = "micron,n25q256a", "jedec,spi-nor";
 		spi-max-frequency = <29000000>;
 		spi-nor,ddr-quad-read-dummy = <6>;
 	};
-- 
2.17.1

